libmscore/cmd.cpp

```diff
@@ -204,41 +204,67 @@ void CmdState::dump()
       }
 #endif
 
+//---------------------------------------------------------
+//   update
```

Comment:
wrong (copy/paste) comment

Reply:
thanks.  I've fixed that...will be included when I next push.